app/controllers/instructor-groups.js

```diff
@@ -18,59 +17,49 @@ export default Controller.extend({
   schoolId: null,
   titleFilter: null,
   showNewInstructorGroupForm: false,
-  newInstructorGroups: [],
-  instructorGroups: computed('selectedSchool', function(){
-    let defer = RSVP.defer();
+  newInstructorGroup: null,
+  deletedInstructorGroups: [],
```

Comment:
Why is one `null` and on and Array?

Reply:
agreed, will change this. also, this doesn't need to be an array at all, since the UI doesn't support bulk deletions at this point. i'm going to change this to be a property that tracks just the most recently deleted instructor group, should be sufficient.